test/support/helpers/stub_helper.rb

```diff
@@ -0,0 +1,24 @@
+module StubHelper
+  extend ActiveSupport::Concern
+
+  included do
+    def stub_const(object, name, val = nil, &block)
```

Comment:
Wow, crazy that this isn't built in to Minitest! Did you explore any ways to do this without these 24 lines of code? Not that they aren't great, but it's a lot "just" to stub a constant.

Maybe we can refactor how the constant is defined or something?

Reply:
Hi, my 2 cents: 

* It seems there isn't such a method in minitest. Batteries are only included in RSpec 😉
* This helper works. However you can always swap it with [minitest-stub-const](https://rubygems.org/gems/minitest-stub-const)
* Since this is a concern, the `included` block is not really needed.